test/ranking/ranking.test.ts

```diff
@@ -0,0 +1,36 @@
+import {Channel} from 'vega-lite/src/channel';
+import {Type} from 'vega-lite/src/type';
+
+import {schema} from '../fixture';
+import {rank} from '../../src/ranking/ranking';
+
+import {SpecQueryModelGroup} from '../../src/modelgroup';
+
+import {assert} from 'chai';
+
+
+describe('ranking', () => {
+  describe('rank', () => {
+    describe('group.items', () => {
```

Comment:
@kanitw: I'm not sure if I've nested these describes properly. Do you have any comments?